packages/winpanda/extra/src/winpanda/docs/README.md

```diff
@@ -1,4 +1,139 @@
-# Winpanda Design Documents
+# Winpanda The DC/OS Windows Package Manager
+
+# Major features
+
+# Documentation
+
+## Command line parametres
+
+
+ ### Usage:
```

Comment:
This line is marked as a header, which should not have a `:` and needs a blank line after it.

Otherwise it renders as:
```
### Usage: ``` winpanda {setup} [options] winpanda {teardown} [options] winpanda {start} [options] winpanda {stop} [options]
```

We probably don't even need this line.